.github/workflows/php.yml

```diff
@@ -68,9 +68,9 @@ jobs:
       - name: "Downgrade dev dependencies"
         run: "composer require phpunit/phpunit:6.5.14 drush/drush:~9 drupal/core-recommended:${{ matrix.drupal }} drupal/core-dev:${{ matrix.drupal }} --with-all-dependencies --dev --no-update"
         if: ${{ matrix.drupal == '^8.9' }}
-      - name: "Upgrade to drupal/core:10.0.x and drush/drush:^11.0"
+      - name: "Upgrade to drupal/core:${{ matrix.drupal }} and drush/drush:^11.0"
         run: "composer require drupal/core-recommended:${{ matrix.drupal }} --with-all-dependencies --dev --no-update"
-        if: ${{ matrix.drupal == '10.0.x-dev' || matrix.drupal == '10.1.x-dev' }}
+        if: ${{ matrix.drupal == '10.0.x-dev' || matrix.drupal == '11.x-dev' }}
```

Comment:
I _think_ `10.0.x-dev` needs to specify `^10.0`. Or if this passed... I don't know!